modules/boot/src/utils/link.cpp

```diff
@@ -45,7 +45,12 @@ KEEP_FUNC void GZ_displayLinkInfo() {
         snprintf(link_x, sizeof(link_x), "pos x: %.4f", player->current.pos.x);
         snprintf(link_y, sizeof(link_y), "pos y: %.4f", player->current.pos.y);
         snprintf(link_z, sizeof(link_z), "pos z: %.4f", player->current.pos.z);
-        snprintf(link_angle, sizeof(link_angle), "rot y: 0x%04X", (u16)player->shape_angle.y);
+        if (g_angle == 0) {
+            snprintf(link_angle, sizeof(link_angle), "rot y: 0x%04X", (u16)player->shape_angle.y);
+        }
+        if (g_angle == 1) {
+            snprintf(link_angle, sizeof(link_angle), "rot y: %d", player->shape_angle.y);
+        }
```

Comment:
after the above is addressed:
```suggestion
        if (g_angleValuesInDecimal) {
            snprintf(link_angle, sizeof(link_angle), "rot y: %d", player->shape_angle.y);
        } else {
            snprintf(link_angle, sizeof(link_angle), "rot y: 0x%04X", (u16)player->shape_angle.y);
        }
```